app/views/volunteers/_form.html.erb

```diff
@@ -0,0 +1,97 @@
+                    <h5>Email:</h5><%= f.text_field :email %>
+                </div>            
+                <div>
+                    <h5>Phone:</h5><%= f.telephone_field :mobile_phone %>
+                </div>            
+                <div>
+                    <h5>City:</h5><%= f.text_field :city %>
+                </div>            
+                <div>
+                    <h5>State:</h5><%= f.text_field :state %>
+                </div>            
+            </div>
+        <div>
+            <h4>Personal Information</h4>
+        </div>
+                <div>
+                    <h5>Birthdate:</h5><%= f.date_field :birthdate %>
+                </div>
+                <div>
+                    <h5>Gender:</h5><p>To which gender identity do you most identify with?</p>
+                    <%= f.select(:gender, options_for_select([
+                        ['Female', 'Female'],
+                        ['Male', 'Male'],
+                        ['Other', 'Other'],
+                        ['Prefer Not to Answer', 'Prefer Not to Answer']])) %>
+                </div>
+                <div>
+                    <h5>Employer</h5><%= f.text_field :employer %>
+                </div>
+                <div>
+                    <h5>Position</h5><%= f.text_field :position %>
+                </div>
+                <div>
+                    
+                </div>
+                <div>
+        <div>
+            <h4>Volunteer Interests and Availability</h4>
+        </div>
+                <div>
+                    <h5>Availability. Select all that apply.</h5>
+                        <!-- style to make multiple checkbox options and style into columns -->
+                        <div>
+                            <%= f.radio_button :availability, 'Monday' %>
+                            <%= f.label :availability, 'Monday' %>
+                        </div>            
+                        <div>
+                            <%= f.radio_button :availability, 'Tuesday' %>
+                            <%= f.label :availability, 'Tuesday' %>
+                        </div>
+                        <div>
+                            <%= f.radio_button :availability, 'Wednesday' %>
+                            <%= f.label :availability, 'Wednesday' %>
+                        </div>
+                        <div>
+                            <%= f.radio_button :availability, 'Thursday' %>
+                            <%= f.label :availability, 'Thursday' %>
+                        </div>
+                        <div>
+                            <%= f.radio_button :availability, 'Friday' %>
+                            <%= f.label :availability, 'Friday' %>
+                        </div>  
+                        <div>
+                            <%= f.radio_button :availability, 'Saturday' %>
+                            <%= f.label :availability, 'Saturday' %>
+                        </div>  
+                </div>
```

Comment:
I think that these might be better as checkboxes, but I'm not confident in what that syntax looks like. Just throwing out that idea if you'd like to look into it.